From 86d3eb547f2d49f87b060441823b9b1ce7927ea2 Mon Sep 17 00:00:00 2001
From: mark-bst <zeqing.xu@bluestacks.com>
Date: Mon, 3 Aug 2026 20:16:43 +0800
Subject: [PATCH] [A16] Hide portrait quick settings header

Port A13 be5549e4c7be after the Android 16 SystemUI layout refactor. The old qs_container no longer exists, so apply its hidden visibility to the remaining QuickQSPanel.
---
 .../SystemUI/res/layout/quick_status_bar_expanded_header.xml     | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/SystemUI/res/layout/quick_status_bar_expanded_header.xml b/packages/SystemUI/res/layout/quick_status_bar_expanded_header.xml
index 1749ed403c09..b0228bf2e370 100644
--- a/packages/SystemUI/res/layout/quick_status_bar_expanded_header.xml
+++ b/packages/SystemUI/res/layout/quick_status_bar_expanded_header.xml
@@ -35,6 +35,7 @@
 
     <com.android.systemui.qs.QuickQSPanel
         android:id="@+id/quick_qs_panel"
+        android:visibility="gone"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginTop="@dimen/qqs_layout_margin_top"
-- 
2.34.1

